helga_jeopardy.py

```diff
@@ -165,37 +214,82 @@ def retrieve_question(client, channel):
 
     """
 
-    logger.debug('initiating question retrieval')
-
-    try:
-        tb_resp = requests.get('{}random.json'.format(api_endpoint))
-    except RequestException:
-        return "Could not retrieve a question from the jservice API"
-
-    json_resp = tb_resp.json()[0]
-    question_text = json_resp['question']
-    answer = json_resp['answer']
-    category = json_resp['category']['title']
-    value = json_resp['value']
-
-    if DEBUG:
-        logger.debug(u'psst! the answer is: {}'.format(answer))
+    if random is True:
+        logger.debug('initiating question retrieval')
+    
+        try:
+            tb_resp = requests.get('{}random.json'.format(api_endpoint))
```

Comment:
I'm pretty sure `tb` stands for `TrivialBuzz`, so this would be a perfect opportunity to rename this variable to something more meaningful like `api_resp`.

```suggestion
            api_resp = requests.get('{}random.json'.format(api_endpoint))
```